nowledge-mem-hermes/provider.py

```
@@ -595,11 +595,11 @@ def save_config(self, values: Dict[str, Any], hermes_home: str) -> None:
         existing: dict = {}
         if config_path.exists():
             try:
-                existing = json.loads(config_path.read_text())
+                existing = json.loads(config_path.read_text(encoding="utf-8"))
             except Exception:
                 pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)